script.js

```diff
@@ -0,0 +1,76 @@
+/**
+ * An octoherd script to watch, unwatch or ignore a repository
+ *
+ * @param {import('@octoherd/cli').Octokit} octokit
+ * @param {import('@octoherd/cli').Repository} repository
+ * @param {{unwatch?: boolean, ignore?: boolean}} options Custom user options passed to the CLI
+ */
+export async function script(octokit, repository, { unwatch, ignore }) {
+	if (unwatch && ignore) {
+		octokit.log.error("Not possible tu unsubscribe and subscribe ignoring notifications")
+		return;
+	}
+	
+	const method = unwatch ? "DELETE" : "PUT";
+	const id = repository.id;
+	const owner = repository.owner.login;
+	const repo = repository.name;
+	// https://docs.github.com/en/rest/reference/activity#get-a-repository-subscription
+	const { subscribed, ignored } = await octokit
+		.request("GET /repos/{owner}/{repo}/subscription", {
+			owner,
+			repo,
+		})
+		.then(
+			({ data: { subscribed, ignored } }) => ({ subscribed, ignored }),
+			() => ({ subscribed: false, ignored: false })
+	);
+	
+	octokit.log.debug(
+		{
+			change: 0,
+			id,
+			owner,
+			repo,
```

Comment:
I thought I already added `owner`, `repo` and repository `id` to all logs within a script by default, but seems like it got lost. I've added the `octokit.log.setContext()` method to `@octoherd/octokit` but didn't implement it in the CLI yet, I'll make an issue as a reminder. But basically you don't need to add this information yourself.